tests/functional/conftest.py

```diff
@@ -23,10 +23,10 @@
 from ape.types import AddressType, ContractLog
 
 
-def _get_raw_contract(name: str) -> Dict:
+def _get_raw_contract(name: str) -> str:
     here = Path(__file__).parent
     contracts_dir = here / "data" / "contracts" / "ethereum" / "local"
-    return json.loads((contracts_dir / f"{name}.json").read_text())
+    return (contracts_dir / f"{name}.json").read_text()
```

Comment:
I don't hate this, but I'm just curious why. Is it easier to deal with strings and the parse_obj here?

Reply:
It just saves a step I guess.

Before we were:

1. Reading the file,
2. Parsing it into a dict 
3. and then parsing that into a `ContractType`.

Now, we:

1. Read the file 
2. and parse it into a `ContractType`

Reply:
Works for me!